odbc/src/tests.rs

```diff
@@ -463,3 +465,401 @@ mod get_diag_rec {
         );
     }
 }
+mod env_attributes {
+    use crate::{
+        definitions::*,
+        handles::{Env, EnvState, MongoHandle},
+        tests::UNIMPLEMENTED_FUNC,
+        SQLGetDiagRecW, SQLGetEnvAttrW, SQLSetEnvAttrW,
+    };
+    use odbc_sys::{
+        AttrConnectionPooling, AttrCpMatch, AttrOdbcVersion, EnvironmentAttribute, HEnv,
+        HandleType, Pointer, SqlReturn,
+    };
+    use std::{ffi::c_void, sync::RwLock};
+
+    const OPTIONAL_VALUE_CHANGED: &str = "01S02\0";
+
+    #[test]
+    fn get_set_odbc_version() {
```

Comment:
Have you tried to write a helper for these tests?
The logic (and most of the code) is the same for all of them.
Having a helper would:
- Allow you to have a concise and logical "test" that can be applied to all the attributes and this way you can make sure the same things are tested for all of them and nothing is forgotten in one of the [test].
- Make things easier when you add a new test case (like testing string_length is null).

Reply:
Done -- it's a little odd because I had to convert every attribute into an `i32` in order to use a helper that works for any environment attribute, but it does make the testing code much cleaner!